packages/electron/lib/install.js

```diff
@@ -25,44 +23,66 @@ module.exports = {
   // returns icons package so that the caller code can find
   // paths to the icons without hard-coding them
   icons () {
-    return require('@cypress/icons')
+    return require('@packages/icons')
   },
 
   checkCurrentVersion () {
     const pathToVersion = paths.getPathToVersion()
 
     // read in the version file
-    return fs.readFileAsync(pathToVersion, 'utf8')
+    return fs.readFile(pathToVersion, 'utf8')
     .then((str) => {
       const version = str.replace('v', '')
 
       // and if it doesn't match the electron version
       // throw an error
       if (version !== electronVersion) {
         throw new Error(`Currently installed version: '${version}' does not match electronVersion: '${electronVersion}`)
-      } else {
-        return process.exit()
+      }
+    })
+  },
+
+  getFileHash (filePath) {
+    return fs.readFile(filePath).then((buf) => {
+      const hashSum = crypto.createHash('sha1')
+
+      hashSum.update(buf)
+      const hash = hashSum.digest('hex')
+
+      return hash
+    })
+  },
+
+  checkIconVersion () {
```

Comment:
Why do we need to update this? I see we are doing something with caching but why is this necessary just to change some icons?

Reply:
On local, electron app is recompiled only if electron version changes or if the app is completely absent. 
If icon is updated, no contributor will see the new icons unless they delete the `dist` directory manually.

I compare the icon in dist with the one in the icons package and if different, force the re-build as well.

Reply:
This seems like a lot of code that has to be maintained for something that just requires devs to `git clean -xfd` once to bypass. I get why you added this, I think, but it's going to be dead code for 99.99% of runs and additional maintenance overhead, I think we could do without these `checkIconVersion` changes.